src/huggingface_hub/hub_mixin.py

```diff
@@ -351,7 +419,7 @@ def __init__(self, *args, **kwargs):
         ...     "mymodel", push_to_hub=False
         >>> )  # Saving model weights in the directory
         >>> model.push_to_hub(
-        ...     "mymodel", "model-1"
+        ...     repo_id="mymodel", commit_message="model-1"
```

Comment:
Is this the only place where this needs to be changed? There might be some documentation in `docs/` that also need to be updated to reflect the new API

Reply:
Actually I went though the documentation and it doesn't seem to be mentioned elsewhere.
In fact, `push_to_hub` mixins possibilities are documented [on this page](https://huggingface.co/docs/huggingface_hub/main/en/package_reference/mixins) which is more a package reference page.